python/pyspark/sql/tests.py

```diff
@@ -3567,6 +3580,18 @@ def tearDownClass(cls):
         time.tzset()
         ReusedSQLTestCase.tearDownClass()
 
+    @property
+    def random_udf(self):
```

Comment:
Could we add "nondeterministic" in its name somehow?

Reply:
Maybe `nondeterministic_udf`. So we don't have duplicate name to `random_udf` too.`